server.py

```diff
@@ -44,6 +56,16 @@ def download(path):
 def server_static(filepath):
     return static_file(filepath, root=os.path.join(dir_path, 'static'))
 
+@app.get('/ranking')
+def rankings():
+	people=[]
+	for u_name in usernames.keys():
+		people.append([u_name,usernames[u_name].getScore(),0])
+	people.sort(key=lambda x: x[1],reverse=True)
+	# rank
+	for i in range(len(people)):
```

Comment:
are you trying to generate ranks here? You could use [enumerate](https://docs.python.org/3.7/library/functions.html#enumerate) which makes it much more readable.

For example, if `people` did not contain ranks originally.

`people = [(name, user, rank) for rank, (name, user) in enumerate(people, start=1)]`

Reply:
I changed the code after sorting. I think it will be fine.